code/modules/events/blob.dm

```diff
@@ -24,11 +23,4 @@
 	B.key = M.key
 	to_chat(B, "<span class='userdanger'>You are now a mouse, infected with blob spores. Find somewhere isolated... before you burst and become the blob! Use ventcrawl (alt-click on vents) to move around.</span>")
 	var/image/alert_overlay = image('icons/mob/blob.dmi', "blank_blob")
-	notify_ghosts("Infected Mouse has appeared in [get_area(B)].", source = B, alert_overlay = alert_overlay)
-
-/datum/event/blob/tick()

```

Comment:
This tick() had to be replaced, because, with mouseblobs, there isn't guaranteed to be a core for a period of time.